test/unit/test-layout.js

```diff
@@ -613,3 +615,91 @@ describe('Layout', () => {
     expect(clone.querySelectorAll('i-amphtml-sizer')).to.have.length(1);
   });
 });
+
+describes.realWin('Layout: aspect-ratio CSS', {amp: true}, function (env) {
+  let win, doc;
+  let element;
+  let ssrSizer;
+
+  beforeEach(() => {
+    win = env.win;
+    doc = win.document;
+    toggleExperiment(win, 'layout-aspect-ratio-css', true, true);
+    resetShouldUseAspectRatioCssForTesting();
+
+    element = doc.createElement('amp-element');
+    element.classList.add('i-amphtml-element');
+    doc.body.appendChild(element);
+
+    ssrSizer = doc.createElement('i-amphtml-sizer');
+    ssrSizer.classList.add('i-amphtml-disable-for-ar');
+  });
+
+  afterEach(() => {
+    resetShouldUseAspectRatioCssForTesting();
+  });
+
+  describe('aspect-ratio not supported', function () {
+    before(function () {
+      if (CSS.supports('aspect-ratio: 1/1')) {
+        this.skipTest();
```

Comment:
Possible to ensure this test is run on something that doesn't support `aspect-ratio` as well?

I believe we support a method to ensure a test runs on ie11 for instance.

Reply:
Could you pls clarify? I'm not familiar with this mechanism. But FWIW, no browser that we use on Travis support aspect-ratio yet. I tested it locally on Chromium though.

Reply:
@rsimha How do we opt-in a specific test for running in older browsers?

Reply:
IE is the only browser on which tests are opt-in. Here's how you can run a test on IE:

https://github.com/ampproject/amphtml/blob/a702b776a7807d446568b0e2309fcd2998c92319/test/integration/test-amp-img.js#L157-L162

Reply:
Thanks! And done!